session/src/main/scala/akka/cassandra/session/DriverConfigLoaderFromConfig.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cassandra.session
+
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.CompletionStage
+
+import com.datastax.oss.driver.api.core.config.DriverConfig
+import com.datastax.oss.driver.api.core.config.DriverConfigLoader
+import com.datastax.oss.driver.api.core.context.DriverContext
+import com.datastax.oss.driver.internal.core.config.typesafe.TypesafeDriverConfig
```

Comment:
Not great to use an internal class, but I see no alternative. Long term we could see if something like this could be added upstream.